terminus-ui/logo/src/logo.component.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { Component, ViewChild } from '@angular/core';
+import {
+  ComponentFixture,
+  TestBed,
+  TestModuleMetadata,
+} from '@angular/core/testing';
+import { configureTestBedWithoutReset } from '@terminus/ngx-tools/testing';
+import { By } from '@angular/platform-browser';
+
+import { TsLogoComponent } from './logo.component';
+import { TsLogoModule } from './logo.module';
+
+
+@Component({
+  template: `
+    <ts-logo id="one" #one></ts-logo>
+    <ts-logo id="two" #two [type]="type" [logoColor]="logoColor"></ts-logo>
+  `,
+})
+class TestHostComponent {
+  type = '';
+  logoColor = '';
+
+  @ViewChild('one')
+  one!: TsLogoComponent;
+
+  @ViewChild('two')
+  two!: TsLogoComponent;
+}
+
+
+
+describe(`TsLogoComponent`, function() {
+  let fixture: ComponentFixture<TestHostComponent>;
+  let hostComponent: TestHostComponent;
+  let logo1: TsLogoComponent;
+  let logo2: TsLogoComponent;
+  const moduleDefinition: TestModuleMetadata = {
+    imports: [
+      TsLogoModule,
+    ],
+    declarations: [
+      TestHostComponent,
+    ],
+  };
+
+  configureTestBedWithoutReset(moduleDefinition);
+
+  beforeEach(() => {
+    // Reset parent component inputs
+    if (hostComponent) {
+      hostComponent.type = '';
+      fixture.detectChanges();
+    }
+
+    fixture = TestBed.createComponent(TestHostComponent);
```

Comment:
probably want to use `createComponent` test helper that got released in ngx-tools 6.1.0, so you don't have to new up TestBed and all. Could also take a look at input component spec for example.